src/main/java/com/epam/test_generator/controllers/SuitController.java

```diff
@@ -132,14 +140,22 @@
         @ApiResponse(code = 400, message = "Invalid input", response = ValidationErrorsDTO.class),
         @ApiResponse(code = 404, message = "Suit not found")
     })
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "projectId", value = "ID of project",
+                    required = true, dataType = "long", paramType = "path"),
+            @ApiImplicitParam(name = "Authorization", value = "add here your token",
+                    paramType = "header", dataType = "string", required = true)
+    })
     @Secured({"ROLE_ADMIN", "ROLE_TEST_ENGINEER", "ROLE_TEST_LEAD"})
-    @RequestMapping(value = "/projects/{projectId}/suits/updateRowNumbers", method = RequestMethod.PUT,
-        consumes = "application/json")
-    @ApiImplicitParam(name = "Authorization", value = "add here your token", paramType = "header", dataType = "string", required = true)
-    public ResponseEntity<List<SuitRowNumberUpdateDTO>> updateSuitRowNumber
-        (@RequestBody @Valid List<SuitRowNumberUpdateDTO> rowNumberUpdates) {
-
-        List<SuitRowNumberUpdateDTO> updatedSuitRowNumberUpdateDTOs = suitService.updateSuitRowNumber(rowNumberUpdates);
+    @RequestMapping(value = "/projects/{projectId}/suits/updateRowNumbers",
+            method = RequestMethod.PUT, consumes = "application/json")
+    public ResponseEntity<List<SuitRowNumberUpdateDTO>> updateSuitRowNumber(
+        @PathVariable("projectId") long projectId,
+             @RequestBody @Valid List<SuitRowNumberUpdateDTO> rowNumberUpdates) {
+
+        final List<SuitRowNumberUpdateDTO> updatedSuitRowNumberUpdateDTOs =
+            suitService
+            .updateSuitRowNumber(projectId,rowNumberUpdates);
```

Comment:
apply codestyle for changeset. (comma must be followed by space character)

`suitService.updateSuitRowNumber(projectId,rowNumberUpdates);` can be written is single row